docs/csharp/quick-starts/interpolated-strings-local.md

```diff
@@ -60,49 +73,46 @@ public class Vegetable
    public override string ToString() => Name;
 }
 
-public class Example
+public class Program
 {
-   public enum Unit { item, pound, ounce, dozen };
-   
    public static void Main()
    {
       var item = new Vegetable("eggplant");
       var date = DateTime.Now;
       var price = 1.99m;
-      var unit = Unit.item;
-      Console.WriteLine($"On {date}, the price of {item} was {price} per {unit}.");
+      Console.WriteLine($"On {date}, the price of {item} was {price} per item.");
```

Comment:
I do like the example that adds the `enum` field value. That's useful for new developers.